tests/e2e/contact-summary.js

```diff
@@ -161,97 +163,64 @@ describe('Contact summary info', () => {
-    helper.waitElementToPresent(element(by.css('#contacts-list')));
+  const selectContact = async term => {
+    await helper.waitUntilReadyNative(contactsPo.searchBox);
+    await contactsPo.searchBox.sendKeys(term);
+    await helper.clickElementNative(contactsPo.searchButton);
+    await helper.waitUntilReadyNative(contactsPo.contactContent());
+    await helper.clickElementNative(contactsPo.contactContent());
+    await helper.waitUntilReadyNative(contactsPo.contactsList());
   };
 
-  it('contact summary', () => {
+  it('contact summary', async () => {
     //disabled.
-    helper.clickElement(element(by.css('#contacts-tab')));
+    await helper.waitUntilReadyNative(contactsPo.contactsTab);
+    await helper.clickElementNative(contactsPo.contactsTab);
     try {
-      selectContact('carol');
+      await selectContact('carol');
     } catch (err) {
-      browser.refresh();
-      helper.waitForAngularComplete();
-      helper.clickElement(element(by.css('#contacts-tab')));
-      selectContact('carol');
+      await browser.refresh();
+      await helper.clickElementNative(contactsPo.contactsTab);
+      await selectContact('carol');
     }
     // assert the summary card has the right fields
-    helper.waitElementToPresent(
-      element(
-        by.css('.content-pane .meta > .card .col-sm-3:nth-child(1) label')
-      )
-    );
-    expect(
-      helper.getTextFromElement(
-        element(
-          by.css('.content-pane .meta > .card .col-sm-3:nth-child(1) label')
-        )
-      )
-    ).toBe('test.pid');
-    expect(
-      helper.getTextFromElement(
-        element(by.css('.content-pane .meta > .card .col-sm-3:nth-child(1) p'))
-      )
-    ).toBe(CAROL.patient_id);
-    expect(
-      helper.getTextFromElement(
-        element(
-          by.css('.content-pane .meta > .card .col-sm-3:nth-child(2) label')
-        )
-      )
-    ).toBe('test.sex');
-    expect(
-      helper.getTextFromElement(
-        element(by.css('.content-pane .meta > .card .col-sm-3:nth-child(2) p'))
-      )
-    ).toBe(CAROL.sex);
+    
+    await helper.waitUntilReadyNative(contactsPo.cardFieldLabel('test_pid'));
+    expect(await contactsPo.cardFieldLabelText('test_pid')).toBe('test_pid');
+    expect(await contactsPo.cardFieldText('test_pid')).toBe(CAROL.patient_id);
+    
+    expect(await contactsPo.cardFieldLabelText('test_sex')).toBe('test_sex');
+    expect(await contactsPo.cardFieldText('test_sex')).toBe(CAROL.sex);
 
-    expect(helper.getTextFromElement(element(by.css('.content-pane .meta > .card .col-sm-3:nth-child(3) label'))))
-      .toBe('aliceTag');
-    expect(helper.getTextFromElement(element(by.css('.content-pane .meta > .card .col-sm-3:nth-child(3) p'))))
-      .toBe(ALICE.name + ' ' + ALICE.phone);
+    expect(await contactsPo.cardFieldLabelText('alicetag')).toBe('aliceTag');
+    expect(await contactsPo.cardFieldText('alicetag')).toBe(ALICE.name + ' ' + ALICE.phone);
```

Comment:
I know it's preexisting stuff, but please update the syntax on this and the others below while you're running thought this bit of code.
```suggestion
    expect(await contactsPo.cardFieldText('alicetag')).toBe(`${ALICE.name} ${ALICE.phone}`);
```